extensions/wikia/Helios/HelperController.class.php

```diff
@@ -26,14 +26,15 @@ public function checkAntiSpoof()
 
 		// Allow the given user name if the AntiSpoof extension is disabled.
 		if ( empty( $this->wg->EnableAntiSpoofExt ) ) {
-			return $this->response->setVal( 'allow', true );
+			$this->response->setVal( 'allow', true );
+			return;
 		}
 
 		$oSpoofUser = new \SpoofUser( $sName );
-		$aConflicts = $oSpoofUser->getConflicts();
```

Comment:
AFAIU this is redundant because the same method is called again a few lines below while the $aConflicts is never used.